helm/values-dev.yaml

```
@@ -112,6 +112,8 @@ microservice-chart:
     REDIS_PASSWORD: redis-ecommerce-access-key
     REDIS_HOST: redis-ecommerce-hostname
     NODO_CONNECTION_STRING: nodo-connection-string
+    NODO_CHECKPOSITION_API_KEY: nodo-nodeforpm-api-key
```

```suggestion
    NODO_NODEFORPM_API_KEY: nodo-nodeforpm-api-key
```

just to have the same naming convention for both the api keys. please refactor application properties too

---

done